gatsby-node.js

```diff
@@ -1,194 +1,26 @@
-// Based upon https://github.com/gatsbyjs/gatsby/tree/master/examples/using-i18n
-exports.onCreatePage = ({ page, actions }) => {
-    const { createRedirect, createPage, deletePage } = actions
-
-    // First delete the incoming page that was automatically created by Gatsby
-    // So everything in src/pages/
-    deletePage(page)
-    const is_responsible_trading = /responsible/g.test(page.path)
-    const is_contact_us = /contact_us/g.test(page.path)
-    const is_careers = /careers/g.test(page.path)
-    const is_p2p = /responsible/g.test(page.path)
-    const who_we_are = /who-we-are/g.test(page.path)
-    const is_cfds = /cfds/g.test(page.path)
-    const is_deriv_ez = /deriv-ez/g.test(page.path)
-    const is_options = /options/g.test(page.path)
-
-    if (is_careers) {
-        createRedirect({
-            fromPath: `/careers/people-management`,
-            toPath: `/careers/human-resources`,
-            redirectInBrowser: true,
-            isPermanent: true,
-        })
-        createRedirect({
-            fromPath: `/careers/people-management/`,
-            toPath: `/careers/human-resources/`,
-            redirectInBrowser: true,
-            isPermanent: true,
-        })
-        createRedirect({
-            fromPath: `/careers/besquare/`,
-            toPath: `/careers/`,
-            redirectInBrowser: true,
-            isPermanent: true,
-        })
-        createRedirect({
-            fromPath: `/careers/besquare`,
-            toPath: `/careers/`,
-            redirectInBrowser: true,
-            isPermanent: true,
-        })
-    }
-
-    if (is_responsible_trading) {
-        createRedirect({
-            fromPath: `/responsible-trading/`,
-            toPath: `/responsible/`,
-            redirectInBrowser: true,
-            isPermanent: true,
-        })
-        createRedirect({
-            fromPath: `/responsible-trading`,
-            toPath: `/responsible`,
-            redirectInBrowser: true,
-            isPermanent: true,
-        })
-    }
-
-    if (is_contact_us) {
-        createRedirect({
-            fromPath: `/contact-us/`,
-            toPath: `/contact_us/`,
-            redirectInBrowser: true,
-            isPermanent: true,
-        })
-        createRedirect({
-            fromPath: `/contact/`,
-            toPath: `/contact_us/`,
-            redirectInBrowser: true,
-            isPermanent: true,
-        })
-        createRedirect({
-            fromPath: `/contact-us`,
-            toPath: `/contact_us`,
-            redirectInBrowser: true,
-            isPermanent: true,
-        })
-    }
+exports.createPages = ({ actions }) => {
+    const { createRedirect } = actions
 
-    if (is_p2p) {
+    redirects.forEach((redirect) => {
         createRedirect({
-            fromPath: `/p2p/v1`,
-            toPath: `/p2p`,
-            redirectInBrowser: true,
+            fromPath: redirect.fromPath,
+            toPath: redirect.toPath,
             isPermanent: true,
```

Comment:
If you want to keep 301 status in the json file can try this
```suggestion
            isPermanent: redirect.status === 301 ? true : false
```